notices.json

```diff
@@ -15,5 +15,23 @@
         "actionParam": "https://mattermost.com/download?inapp-notice=true#mattermostApps"
       }
     }
+  },
+  {
+    "id": "server_upgrade_v5.29",
+    "conditions": {
+      "audience": "sysadmin",
+      "clientType": "all",
+      "serverVersion": ["<5.29"],
+      "displayDate": ">= 2020-11-16T00:00:00Z"
```

Comment:
It happens relatively infrequently these days, but in case the release is delayed due to some last minute major bug fix, or if we have to do a quick dot release directly post-release for some major bug, would it be possible to send this notice a few days after the release, just in case?

Reply:
For sure, I'll change it to the 19th